GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+            | `BUILD_ARTIFACT_NAME` | drop | Name to identity the folder containing artifacts output by a build. |
+            | `GO_VERSION`| 1.12.5 | The version of Go terraform deployments are bound to. |
+            | `PIPELINE_ROOT_DIR` | 'devops/providers/azure-devops/templates/' | A path for finding Cobalt templates. |
+            | `REMOTE_STATE_CONTAINER` | az-hw-remote-state-container | The remote storage container name for managing the state of a cobalt template's deployed infrastructure. Also is used as a naming convention for branching state into multiple workspaces. |
+            | `SCRIPTS_DIR` | infrastructure/scripts | Path to scripts used at runtime for composing build and release jobs at various pipeline stages. |
+            | `TEST_HARNESS_DIR` | test-harness/ | A path to the cobalt test harness for running integration and unit tests written in Docker and Golang. |
+            | `TF_ROOT_DIR`| infra | The primary path for all Cobalt templates and the module they are made of.  |
+            | `TF_VERSION`| 0.12.4 | The version of terraform deployments are bound to. |
+            | `TF_WARN_OUTPUT_ERRORS`| 1 | The severity level of errors to report |
+
+    > Important: Every targeted environment specified within the build pipeline expects a
+    > variable group specified with the naming convention `<ENVIRONMENT_NAME> Pipeline Variables`
+
+    * Configure *DevInt Pipeline Variables*
+        * Environment-specific variables have no default values and must be assigned
+        * Return to the Library tab
+        * Click [+Variable group] and add the following variables:
+
+            | Name  | Description
+            |-------------|-----------|
+            | `ARM_SUBSCRIPTION_ID` | The Azure subscription ID for the DevInt environment to which resources will be deployed |
+            | `REMOTE_STATE_ACCOUNT` | The remote state account name used to manage the state of this environment's deployed infrastructure |
+            | `SERVICE_CONNECTION_NAME` | The name of the service endpoint or connection created in a previous step with the Service Principal permissions to deploy resources to the specified Azure subscription |
+
+    * Configure *QA Pipeline Variables*
+        * Environment-specific variables have no default values and must be assigned
+        * Return to the Library tab
+        * Click [+Variable group] and add the same three variables with QA environment values
+
+    * Link Variable Groups for DevInt, QA and Infrastructure to the Build Pipeline
+        * Select Pipelines tab from within side-navigation menu
+        * Select existing pipeline and then click [Edit]
+        * Next to the [Variables] button at the top of the page, click the ellipses and select Triggers at the top of the page
+        ![Triggers](https://user-images.githubusercontent.com/41071421/63284806-022fda80-c27a-11e9-8e23-494314c63651.png)
+        * Link each variable group, one by one
```

Comment:
Need to describe how to get here (Triggers > Variables > Variable Groups

Reply:
Document now mentions variables tab.